extension.toml

```diff
@@ -0,0 +1,17 @@
+name = "MATLAB"
+version = "0.0.2"
+authors = ["Ruben Zukic <ruben@zukic.net>"]
+description = "MATLAB syntax highlighting for Zed"
+repository = "https://github.com/rzukic/zed-matlab"
+id = "matlab"
+schema_version = 1
+
+[languages]
```

Comment:
Listing the languages in the `extension.toml` is not required and is not recommended, as we read them from the `languages/` directory during the packaging process.